### PR TITLE
refactor: image full token

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  -         |
| 🇨🇳 Chinese |   -        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/image/style/index.tsx [hunk 1]
```diff
@@ -9,29 +9,24 @@ import {
...
-  switchLeft: number;
-  switchRight: number;
-  switchWidth: number;
-  switchHeight: number;
-  switchMarginTop: number;
-  width1px: number;
-  modalMaskBg: string;
+  imagePreviewOperationColor: string;
+  imagePreviewSwitchSize: number;
+  imagePreviewSwitchMargin: number;
```
Comment (zombieJ) [inline components/image/style/index.tsx:21]: 太多了，文字相关的直接从序列里找贴近的选：https://ant.design/docs/spec/font-cn#%E5%AD%97%E9%98%B6%E4%B8%8E%E8%A1%8C%E9%AB%98

在 alias 里应该有对应的列表

Comment (zombieJ) [reply components/image/style/index.tsx:21]: 另外颜色相关的也有 mask 对应的，再改改

**Comment by zombieJ** [comment]
CI 挂了不少

File: components/image/style/index.tsx (hunk 1 above)
Comment (MadCcc) [reply components/image/style/index.tsx:21]: fontSize margin 之类的可以平替，size 这种也有么

**Comment by zombieJ** [comment]
诶。。。还在过。 @MadCcc 开个新的 PR 继续吧

**Pull request merged**